numba_dpex/core/runtime/CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+cmake_minimum_required(VERSION 3.21...3.27 FATAL_ERROR)
+
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.24")
+    cmake_policy(SET CMP0135 NEW)
+endif()
+
+project(_dpexrt_python
+    DESCRIPTION "A Python C extension for numba-dpex runtime."
+)
+
+# Get numba include path
+execute_process(
+    COMMAND python -c "import numba; print(numba.extending.include_path());"
+    OUTPUT_VARIABLE Numba_INCLUDE_DIRS
+    RESULT_VARIABLE RET
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+if(RET EQUAL "1")
+    message(FATAL_ERROR "Module \'numba\' not found.")
+endif()
+
+# Get dpctl library path
+execute_process(
+    COMMAND python -c "import dpctl; import os; print(os.path.dirname(dpctl.__file__));"
+    OUTPUT_VARIABLE DPCTL_LIBRARY_PATH
+    RESULT_VARIABLE RET
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+if(RET EQUAL "1")
+    message(FATAL_ERROR "Module \'dpctl\' not found.")
+endif()
+
+# Update CMAKE_MODULE_PATH
+set(DPCTL_MODULE_PATH ${DPCTL_LIBRARY_PATH}/resources/cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${DPCTL_MODULE_PATH})
+
+# Get scikit-build path
+execute_process(
+    COMMAND python -c "import skbuild; print(skbuild.__path__[0]);"
+    OUTPUT_VARIABLE SKBUILD_PATH
+    RESULT_VARIABLE RET
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+if(RET EQUAL "1")
+    message(FATAL_ERROR "Module \'skbuild\' not found.")
+endif()
+
+# Update CMAKE_MODULE_PATH
+set(SKBUILD_MODULE_PATH ${SKBUILD_PATH}/resources/cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${SKBUILD_MODULE_PATH})
+
+# Check CMAKE_MODULE_PATH
+message(STATUS "CMAKE_MODULE_PATH=" "${CMAKE_MODULE_PATH}")
+
+# Add packages
+find_package(PythonInterp 3)
```

Comment:
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html

Reply:
It's working now, we just had to add `cmake` along with `scikit-build` in the conda environment.